### PR TITLE
Improvement/base template

### PR DESCRIPTION
**Pull request by krysal**
## Description
This PR mainly corrects several accessibility / SEO / performance issues pointed by [Lighthouse](https://web.dev/measure/) test.

## Technical details
- Allow toggling burger menu in navigation bar (appears on screens of the size of mobile devices like phone and tablets)
- Add label for input on footer (for screen readers only)
- Add alternate text to logo image, language attribute to `<html>` tag and a meta description.
- Defers javascript code and specify `media` attribute of css stylesheets

## Deployment preview
See https://caselaw-mirr-improvemen-ouodwv.herokuapp.com/

## Screenshots

Before changes the measures were:
Captura de pantalla de 2020-08-03 01-39-10

and after, the result is:
Captura de pantalla de 2020-08-03 02-06-57

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the ~~*default*~~ *develop* branch of the repository ~~(`main` or `master`).~~
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: legal_db/static/app.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+const burgerMenu = document.getElementsByClassName('navbar-burger')[0];
+burgerMenu.addEventListener('click', () => {
+    const menu = document.getElementsByClassName('navbar-menu')[0];
+    menu.classList.toggle('is-active');
+    burgerMenu.classList.toggle('is-active');
+    if (burgerMenu.classList.contains('is-active')) {
+        burgerMenu.setAttribute('aria-expanded', true);
+    } else {
+        burgerMenu.setAttribute('aria-expanded', false);
+    }
```
Comment (zackkrida) [inline legal_db/static/app.js:11]: how about:

`burgerMenu.setAttribute('aria-expanded', burgerMenu.classList.contains('is-active'));`

😸

File: legal_db/templates/legal_db/base.html [hunk 2]
```diff
@@ -2,16 +2,16 @@
 {% load static %}
 
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (zackkrida) [inline legal_db/templates/legal_db/base.html:5]: 👍

File: legal_db/templates/legal_db/base.html [hunk 3]
```diff
@@ -21,7 +21,7 @@
     {% endblock %}
     {% include 'legal_db/partials/_footer.html' %}
 
-    <script>
+    <script defer>
```
Comment (zackkrida) [inline legal_db/templates/legal_db/base.html:24]: `defer` will have no effect on an inline script block.

File: legal_db/static/app.js (hunk 1 above)
Comment (krysal) [reply legal_db/static/app.js:11]: Better! thanks 😄

**Comment by krysal** [comment]
Thanks for the comments @zackkrida, changes applied!

**Pull request merged**